### PR TITLE
Fixed all kinds of stuff.

### PR DESCRIPTION
**Pull request by BlockWorker**
- random node colors now actually contain blue :D
- new, less ugly power beam
- (somewhat) random color and alpha for every crystal
- basic power beam settings (right click on any node to switch)

http://imgur.com/a/H9rX6

### CODE REVIEW

File: src/main/scala/com/itszuvalex/femtocraft/power/node/PowerNode.scala [hunk 1]
```diff
@@ -25,16 +25,21 @@ trait PowerNode extends TileEntity with IPowerNode {
...
+                                                      (Random.nextInt(30) - 15).toByte,
+                                                      (Random.nextInt(30) - 15).toByte).toInt )
 
   initializePowerSettings()
 
   def initializePowerSettings(): Unit
 
 
-  def savePowerConnectionInfo(compound: NBTTagCompound) = {
+  def savePowerConnectionInfo(compound: NBTTagCompound) =  {
```
Comment (BlockWorker) [inline src/main/scala/com/itszuvalex/femtocraft/power/node/PowerNode.scala:42]: sorry, that was a bit of a fail :P

**Comment by BlockWorker** [comment]
Closed for now, as you are doing stuff there yourself

**Comment by BlockWorker** [comment]
Could you please make a to-do list for me? I don't know what to do exactly, but I want to work on Femtocraft :P
P.S. I post this here because I have no other reliable way of telling you.

**Comment by Itszuvalex** [comment]
There's a very minimal one on the wiki.  If you need to reach me, use ''@Itszuvalex'' in the irc channel and it should ping me through slack.

**Pull request closed without merging**